wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -500,15 +500,20 @@ extension ConversationCreationController {
     }
 }
 
-extension Feature.MLS.Config.MessageProtocol {
-    var toMessageProtocol: MessageProtocol {
-        switch self {
+extension UserSession {
+
+    var defaultProtocol: MessageProtocol {
+        guard mlsFeature.isEnabled else {
+            return .proteus
+        }
+        switch mlsFeature.config.defaultProtocol {
         case .proteus:
-            .proteus
+            return .proteus
         case .mls:
-            .mls
+            return .mls
         case .mixed:
-            .mixed
+            return .mixed
```

Comment:
`.mixed` doesn't make sense as a default protocol. We had two separate message protocol types (one with mixed, one without) for the reason that one represents what the current protocol of a conversation is (which can be mixed) and the other represents what different protocols there are (without mixed).

The value should be either proteus or mls.

Reply:
Changed it to Proteus